recipes/open62541/all/patches/0005-use-external-libxml2.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1092,7 +1092,8 @@ if(UA_ENABLE_NODESETLOADER)
+     list(APPEND lib_sources ${NODESETLOADER_SOURCES})
+     list(APPEND plugin_headers ${PROJECT_SOURCE_DIR}/plugins/include/open62541/plugin/nodesetloader.h)
+     list(APPEND plugin_sources ${PROJECT_SOURCE_DIR}/plugins/ua_nodesetloader.c)
+-    list(APPEND open62541_LIBRARIES ${NODESETLOADER_DEPS_LIBS})
++    find_package(LibXml2 REQUIRED)
```

Comment:
Conan's file name is `libxml2`
```suggestion
+    find_package(libxml2 REQUIRED)
```

Reply:
This was not wrong to begin with, but both work.
The legacy canonical name is `LibXml2`(a built-in find module in cmake), and the libxml2 project uses `libxml2` all lowercase. CMake looks for `<lowercase>-config.cmake` _regardless_ of the name, so `LibXml2` (which is what is more widely in use), will happily match and be found, with the added advatange that if the build scripts evaluate `LibXml2_FOUND`, it will also be defined. Not the case here: both `find_package(LibXml2 REQUIRED)` and  `find_package(libxml2 REQUIRED)` are behaving exactly the same way - or should (once the libxml2 and/or cmakeconfigdeps stop providing find modules)